--- lib/rpmvs.c	2026-02-22 14:10:26.611478260 +0800
+++ lib/rpmvs.c.new	2026-02-22 14:13:59.884000000 +0800
@@ -424,18 +424,8 @@
     for (int i = 0; i < sis->nsigs; i++) {
 	struct rpmsinfo_s *sinfo = &sis->sigs[i];
 	if (sinfo->range & range) {
-	    if (sinfo->rc != RPMRC_OK)
-		continue;
-
-	    rpmDigestBundleAddID(sis->bundle, sinfo->hashalgo, sinfo->id, 0);
-	    /* OpenPGP v6 signatures need a grain of salt to go */
-	    if (sinfo->type == RPMSIG_SIGNATURE_TYPE && sinfo->sig) {
-		const uint8_t *salt = NULL;
-		size_t slen = 0;
-		if (pgpDigParamsSalt(sinfo->sig, &salt, &slen) == 0 && salt) {
-		    rpmDigestBundleUpdateID(sis->bundle, sinfo->id, salt, slen);
-		}
-	    }
+	    if (sinfo->rc == RPMRC_OK)        
+                rpmDigestBundleAddID(sis->bundle, sinfo->hashalgo, sinfo->id, 0);                  
 	}
     }
 }
